src/View/StringTemplate.php

```diff
@@ -323,4 +324,65 @@ protected function _formatAttribute($key, $value, $escape = true)
 
         return $key . '="' . ($escape ? h($value) : $value) . '"';
     }
+
+    /**
+     * Adds a class and returns a unique list either in array or space separated
+     *
+     * ### Options
+     *
+     * - `useIndex` if you are inputting an array with an 'element' called 'class' and want the same returning
+     *                  you should set this to 'class'
+     * - `returnArray` Setting this to `false` will return a space separated string (default is `true`)
+     *
+     * @param array|string $input The array or string to add the class to
+     * @param array|string $newClass the new class or classes to add
+     * @param array $options See above for options
+     * @return array|string
+     */
+    public function addClass($input, $newClass, $options = [])
+    {
+        // NOOP
+        if (empty($newClass)) {
+            return $input;
+        }
+
+        $options = $options + [
```

Comment:
Yes that way it adds defaults and the options take precedence.

I've changed to

```php 
        $options += [
                'useIndex' => 'class',
                'asString' => false
            ];
```